constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordination.scala

```diff
@@ -70,21 +74,10 @@ final class ConsulCoordination(
 
   @volatile var stateSession: Option[SessionId] = None
 
-  private val logger = system.log
-
-  private val host =
-    system.settings.config.getString("constructr.coordination.host")
+  implicit val logger: LoggingAdapter = Logging.getLogger(system, this)
```

Comment:
just because the `log` methods in Akka Streams can take an implicit `LoggingAdapter`, and that would be used as the _logger_. It could be explicitly passed and so this field won't be needed to be declared implicit.

Considering visibility, this could also be made protected and still works, if we feel worried about the visibility of it...